addons/mail/static/src/core/common/attachment_list.scss

```diff
@@ -38,3 +38,14 @@
 .o-viewable {
     cursor: zoom-in;
 }
+
+.o-mail-AttachmentCard-CancelUpload {
+    .fa-times {
+        z-index: 2;
+        font-size: 1.2rem;
+    }
+
+    .fa-circle-o-notch {
+        font-size: 2rem;
+    }
```

Comment:
We don't override bootstrap classes this way. As it would affect other places as well. If you need to do something custom then you create your custom classes and modify them.